protocol/bc/blockheader.go

```diff
@@ -36,3 +36,12 @@ func NewBlockHeaderEntry(version, height uint64, previousBlockID Hash, timestamp
 	bh.Body.NextConsensusProgram = nextConsensusProgram
 	return bh
 }
+
+// checkValid does only part of the work of validating a block. The
+// rest is handled in ValidateBlock, which calls this.
+func (bh *BlockHeaderEntry) checkValid(_ *validationState) error {
```

Comment:
if you want, it's okay to leave out the parameter name entirely when all of the params are unused, e.g.

```go
func (bh *BlockHeaderEntry) checkValid(*validationState)
```